js/AttributeEditor.js

```diff
@@ -0,0 +1,56 @@
+// @flow
+'use strict';
+
+/** Editor for attributes. **/
+class AttributeEditor {
+  onApply: Function;
+  operations: Array;
+
+  constructor(onApply: Function) {
+      this.onApply = onApply;
+      this.operations = [];
+  }
+
+  /**
+   * Adds string attribute.
+   * @instance
+   * @memberof AttributeEditor
+   * @function setString
+   *
+   * @param {string} value The attribute value.
+   * @param {string} name The attribute name.
+   * @return {AttributeEditor} The attribute editor instance.
+   */
+  set(value: string, name: string): AttributeEditor {
+    var operation = { "action": "set", "value": value, "key": name }
+    this.operations.push(operation)
+    return this;
+  }
+
+  /**
+   * Removes the attribute.
+   * @instance
+   * @memberof AttributeEditor
+   * @function removeAttribute
+   *
+   * @param {string} name The name of the attribute to remove.
+   * @return {AttributeEditor} The attribute editor instance.
+   */
+  removeAttribute(name: string): AttributeEditor {
```

Comment:
should this be just `remove` or should we make the set `setAttribute`?

Reply:
changed set to setAttribute